lib/index.ts

```diff
@@ -146,3 +182,39 @@ async function getManifestFiles(
     })
     .filter((i) => i.contents !== "");
 }
+
+async function calcHashOfBinFiles(
+  docker: Docker,
+  files: string[],
+  options?: any,
+): Promise<BinHashFile[]> {
+  const resultArr: BinHashFile[] = [];
+
+  for (const file of files) {
+    let hash: crypto.Hash;
+    if (options && options.hash) {
+      hash = crypto.createHash(options.hash);
+    } else {
+      // defaults to sha1
+      hash = crypto.createHash("sha1");
+    }
```

Comment:
Is this used by some external service? SHA1 is deprecated and shouldn't really be used

Reply:
afaik, sha1 is used by maven central, so I must use it. if that's not the case I will change it to sha-256 (or sha-512)

Reply:
the reasoning here makes sense, but we might want to run this by the internal-security channel at least as a heads up, apparently using sha1 for any reason causes questions

Reply:
![image](https://user-images.githubusercontent.com/16254901/77943115-9c4a0d80-72b4-11ea-8117-873167b6d915.png)

As you can see here, mvn central api is looking for sha1, so I dont think we have an option